src/olympia/versions/tests/test_models.py

```diff
@@ -1144,12 +1146,11 @@ def test_commits_to_git_waffle_enabled(self):
                 parsed_data=parsed_data)
         assert version.pk
 
-        repo = AddonGitRepository(addon.pk)
-        assert os.path.exists(repo.git_repository_path)
+        create_entry_mock.assert_called_once_with(version=version)
 
-    @mock.patch('olympia.versions.tasks.extract_version_to_git.delay')
+    @mock.patch('olympia.git.utils.create_git_extraction_entry')
     @override_switch('enable-uploads-commit-to-git-storage', active=True)
-    def test_commits_to_git_async(self, extract_mock):
+    def test_creates_git_extraction_entry_async(self, create_entry_mock):
```

Comment:
What's the difference between this test case and `test_creates_git_extraction_entry`, above?

Reply:
Nothing. I'll remove one test case.